### DISCUSSION TITLE
Published reference docs for Kube CRDs?

### DISCUSSION THREAD

**Discussion by sgrissom-rxss** [argoproj/argo-cd#14142] [Q&A] [answered]
Are the ArgoCD CRDs rendered to html and published somewhere? The [declarative setup](https://argo-cd.readthedocs.io/en/stable/operator-manual/declarative-setup/) page has links to raw CRDs, but I can't find a reference section where these are rendered in html form. I'm looking for something like [ArgoCD Operator API Reference](https://argocd-operator.readthedocs.io/en/latest/reference/api.html/) or the [Kubernetes API Reference](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.27/).

Edit: Well, the files on the declarative page aren't the CRDs. But the question still stands if html rendered api docs exist somewhere.

**Comment by crenshaw-dev** [comment, marked as answer]
Nothing quite as solid as those API docs. But starting with the 2.8 docs, we'll have HTML renderings of the example YAMLs. You can see them today in the `latest` docs.

* Declarative guide now links to HTML: https://argo-cd.readthedocs.io/en/latest/operator-manual/declarative-setup/
* Project spec: https://argo-cd.readthedocs.io/en/latest/operator-manual/project-specification/
* Application spec: https://argo-cd.readthedocs.io/en/latest/user-guide/application-specification/
* ApplicationSet spec: https://argo-cd.readthedocs.io/en/latest/operator-manual/applicationset/applicationset-specification/

**Reply by sgrissom-rxss** [reply]
Thanks @crenshaw-dev, the rendered html versions of the examples are definitely useful.
